### ISSUE TITLE
Unbale to install httpd service in bash script (CenOS 7)

### ISSUE THREAD

**Issue by SabareeshUniq** [devopshydclub/vprofile-project#219]
Getting error while installing Httpd dervice

**Comment by YagneshTrainer** [comment]
I guess you have to work with rocky linux instead of centos

**Comment by SabareeshUniq** [comment]
Yeah i think there is some problem on **centOS 7** and i tried installing **euroLinux  centOS 9** and i can able to install the Httpd service
